src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -147,6 +147,11 @@ public bool PreferencesDebugMode
             }
         }
 
+        /// <summary>
+        /// Returns all installed packages
+        /// </summary>
+        public ObservableCollection<PackageViewModel> LocalPackages => installedPackagesViewModel.LocalPackages;
```

Comment:
Not sure I understand why this is needed as the one that is bound to in the installed packages control is the `LocalPackages` property belonging to `InstalledPackagesViewModel` and not `PreferencesViewModel`.